pkg/plugins/runtime/k8s/controllers/inbound_converter.go

```diff
@@ -166,6 +168,7 @@ func (i *InboundConverter) LegacyInboundInterfacesFor(ctx context.Context, zone
 	return ifaces, nil
 }
 
+// Should be used when MeshService mode is Exclusive
 func (i *InboundConverter) InboundInterfacesFor(ctx context.Context, zone string, pod *kube_core.Pod, services []*kube_core.Service) ([]*mesh_proto.Dataplane_Networking_Inbound, error) {
```

Comment:
Do we need to sort `services []*kube_core.Service` before iterate over it? To keep consistant results when deduplicated in multiple runs.

Reply:
we sort inbounds at the end